client/src/Core/Components/MainPagesWrapper.tsx

```diff
@@ -40,7 +37,4 @@ const MainPagesWrapperStyles = ({ theme }: IThemeOnlyProps): MainPagesWrapperSty
   ]
 });
 
-const publishControlAreaStyles = ({ theme }: IThemeOnlyProps): Partial<PublishControlAreaStyles> => ({
-  root: [stickyHeaderStyle(theme)]

```

Comment:
what was this piece of code doing? 
Just wanted to know as it is not being used in MainLayout.tsx now

Reply:
It was majorly adding the grey colored bottom-border. I was using a separator for it, as was being used by some other files also. But I think `stickyHeader` would make things more compact => Removing the `Separator` and using `stickyHeader` for the `navAndContorlArea`.